app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
```

Comment:
You can use self. instead of Person.
```suggestion
        self.people[name] = self
```